gcloud/resource_manager/client.py

```diff
@@ -0,0 +1,298 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A Client for interacting with the Resource Manager API.
+
+Overview
+~~~~~~~~
+
+There are three main methods in the ``Client`` class:
+
+- :func:`gcloud.resource_manager.client.Client.list_projects`
+- :func:`gcloud.resource_manager.client.Client.project`
+- :func:`gcloud.resource_manager.client.Client.get_project`
+
+``project()`` versus ``get_project()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The difference between ``project`` and ``get_project`` is subtle,
+so it might be worthwhile to make a quick distinction.
```

Comment:
That takes us back to the original point, the subtlely around what `<noun>()` does. From afar, it's unclear. Once you know what it does, you already know.